daily_post.py

```diff
@@ -5,23 +5,88 @@
 import praw
 import requests
 import time
+import markovify
+from unidecode import unidecode
+import re
+import random
 
 
 from bot import *
 from bs4 import BeautifulSoup
 from jinja2 import Environment, FileSystemLoader
 
+
+STATE_SIZE = 2
+INFO_URL = 'https://github.com/trambelus/UserSim'
+SUB_URL = '/r/User_Simulator'
+SUB = 'SeattleWA'
```

Comment:
Statis configs like this should be placed in the bot.py

`FOOTER` can stay here since it's dynamically generated